src/components/downloadRow.jsx

```diff
@@ -98,7 +99,11 @@ DownloadRow.propTypes = {
   }).isRequired,
   inCart: PropTypes.bool.isRequired,
   onCartClick: PropTypes.func.isRequired,
-  siteName: PropTypes.string.isRequired,
+  siteName: PropTypes.string,
+};
+
+DownloadRow.defaultProps = {
+  siteName: '',
```

Comment:
More of a topic for discussion: 
On the download page, `siteName` was being used to categorize whether or not something that is Pending Q.C is available for a user to download. Would it make sense to add like a `grant` or `group` property since that seems to be where the permissions barrier is likely to be placed? Or do you think this type of permission (allowing users to download) should be handled exclusively on the backend?

Reply:
This is a great question. Personally, I think having more granularity in user's metadata never hurts, such as the _grant(s)_, _group(s)_, or even _role(s)_ associated with a given user. That said, the business logic to handle the permissions based on a given user's role would be more suitable to be implemented on the backend.